doc/local-pv-allocation.md

```diff
@@ -0,0 +1,128 @@
+# Local PV Allocation
+The purpose of this document is to show how to provision a VM disk on a specific node to be able to run it on the same node.
```

Comment:
The purpose of this document is to show how to provision a Virtual Machine(VM) disk image on a specific node to be able to run the VM on the same node. This is important if you are using local-volume storage and you want a disk image to be provisioned on a particular node.